x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -117,7 +117,14 @@ export class TaskManagerPlugin
         usageCollection,
         monitoredHealth$,
         this.config.ephemeral_tasks.enabled,
-        this.config.ephemeral_tasks.request_capacity
+        this.config.ephemeral_tasks.request_capacity,
+        this.config.unsafe.exclude_task_types
+      );
+    }
+
+    if (this.config.unsafe.exclude_task_types.length) {
+      this.logger.debug(
```

Comment:
Since we've labelled this as unsafe/experimental, wondering if we should elevate this message to an info, or perhaps even warning.

Reply:
+1 to a warning. It would make it clear what's happening without a way to ignore it.

Reply:
Good idea!